src/main/java/io/cdap/plugin/gcp/bigquery/source/BigQuerySource.java

```diff
@@ -74,6 +74,7 @@
   private BigQuerySourceConfig config;
   private Schema outputSchema;
   private Configuration configuration;
+  private Boolean deleteBucket;
```

Comment:
Where is this one used?

Reply:
Oops, accidentally commited something from an initial idea. Deleted.